package.json

```diff
@@ -22,6 +22,8 @@
   },
   "devDependencies": {
     "@vitest/coverage-v8": "^0.33.0",
+    "eslint": "^8.52.0",
+    "eslint-plugin-unicorn": "^48.0.1",
```

Comment:
Why is this removed?

If this was intentional, please provide a reason and run `npm install` to update package-lock and push it to repo. This is the reason tests are failing.

Reply:
hello